include/mbedtls/platform.h

```diff
@@ -43,6 +43,9 @@
 #include "platform_time.h"
 #endif
 
+#define MBEDTLS_ERR_PLATFORM_HW_FAILED           -0x0080 /**< Hardware failed platform operation. */
+#define MBEDTLS_ERR_PLATFORM_FEATURE_UNSUPPORTED -0x0082 /**< The requested feature is not supported by the platform */
```

Comment:
The value probably needs to be changed.
I get the following:
~~~
programs/util/strerror 0x82
Last error was: -0x0082 - UNKNOWN ERROR CODE (0080) : BIGNUM - An error occurred while reading from or writing to a file
~~~
I'll change to 0x78, assuming the parsing will work

Reply:
The problem is that our low level error codes are defined as:
~~~
(0x0002-0x007E, 0x0003-0x007F)
~~~